index.Js

```diff
@@ -0,0 +1,25 @@
+//here we are going to define and declare object and inside the object I define two arrays
+const object = {
+  input:['Hello','How are you?','What is your favourite colour?'],
+  output:['Hi','Great!','I have so many favorites it hard to choose one']
+  };
+  console.log(object);
+//here we define function called reply and this function will let the chatpot to give me the proper anwers based on the input the we give
+  function reply(){
```

Comment:
We haven't talked about it so much in class, but generally it helps to improve readability in your code if you follow these rules for adding whitespace: [see here](https://www.crockford.com/code.html). You should be able to format your code with your code editor so you don't have to add the spacing manually. 

For example, here's your reply function with spacing added:
```javascript
  function reply() {
    //I defince variable called question and I put inside this variable the value (input from user)
    let question = document.getElementById('input').value;
    //I used if statement to check the input and give the outpot
    if (object.input.includes(question)) {
      //here we define variable called i, inside this variable i checked the index of the input based on the location in array
      let i = object.input.indexOf(question);
      //last step is to give the output location based on the input location in the array
      document.getElementById('output').textContent = object.output[i];

    } else {
      document.getElementById('output').textContent = "I don't understand that command. Please enter another";

    }
  }
```